File: src/ios/Logger.m
#import "Logger.h"

@implementation Logger

- (void) setLoggingEnabled:(CDVInvokedUrlCommand *) command{

    isLoggingEnabled = [command.arguments objectAtIndex:0];
    NSLog(@"[Logger] Enabled : %@",isLoggingEnabled ? @"True" : @"False");

}

- (void) printLog: (CDVInvokedUrlCommand *) command {

    if (isLoggingEnabled){
        NSString *message = [command.arguments objectAtIndex:0];
        NSLog(@"[Logger] %@",message);
    }
}

@end

File: www/Logger.js
var exec = require('cordova/exec');

var PLUGIN_NAME = "Logger";

var Logger = {

    setLoggingEnabled: function(isLoggingEnabled){
        exec(null, null, PLUGIN_NAME, 'setLoggingEnabled', [isLoggingEnabled]);
    },

    printLog: function(message){
        exec(null, null, PLUGIN_NAME, 'printLog', [message]);
    }

};

module.exports = Logger;